crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -264,6 +275,7 @@ impl<'a> NextFinishedBundle<'a> {
 
 /// The size of the `seq_action` in bytes, including the rollup id.
 fn estimate_size_of_sequence_action(seq_action: &SequenceAction) -> usize {
+    const FEE_ASSET_ID_LEN: usize = 32;
```

Comment:
Note here: this is likely wrong now - and also unnecessary. We know the the byte-length of the asset string.

Reply:
the max size should be 36 (32-byte id plus `ibc/` prefix`), if the denom string is >32 bytes it must get converted to the `ibc/` prefixed form

Reply:
The effective size is actually 68 bytes: hex-formatting doubles the size of 32 bytes to 64 (+4 due to `ibc/`). We need to decide if we prefer legibility or saving 36 bytes (note that from the PoV of sequencer-relayer this likely does not matter due to compression).

Reply:
I have addressed this in [981930e](https://github.com/astriaorg/astria/pull/1209/commits/981930ef11dfc3f1db2501ef54b016fbb91d4888).

The size estimator overall was silly: we can use `prost::Message::encoded_len` to get the actual true length of the action as it becomes a protobuf. (there is a non-zero cost to go `sequence action -> proto sequence action -> proto len`, but that's likely negligible for a service like composer).